projects/1.16/assets/fantasys-furniture/fantasyfurniture/lang/zh_cn.json

```
@@ -0,0 +1,171 @@
+  "block.fantasyfurniture.decorations.coin_stack_gold": "金币堆",
+  "block.fantasyfurniture.decorations.coin_stack_iron": "铁币堆",
+  "block.fantasyfurniture.decorations.dunmer.pottery_0": "丹莫陶器 1",
+  "block.fantasyfurniture.decorations.dunmer.pottery_1": "丹莫陶器 2",
+  "block.fantasyfurniture.decorations.muffins_blueberry": "蓝莓松饼",
+  "block.fantasyfurniture.decorations.muffins_blueberry.stacked": "堆积的蓝莓",
+  "block.fantasyfurniture.decorations.muffins_chocolate": "巧克力松饼",
+  "block.fantasyfurniture.decorations.muffins_chocolate.stacked": "堆积的巧克力",
+  "block.fantasyfurniture.decorations.muffins_sweetberry": "甜莓松饼",
+  "block.fantasyfurniture.decorations.muffins_sweetberry.stacked": "堆积的甜莓",
+  "block.fantasyfurniture.decorations.mushrooms_red": "红蘑菇",
+  "block.fantasyfurniture.decorations.mushrooms_red.stacked": "堆积的红蘑菇",
+  "block.fantasyfurniture.decorations.nordic.boiled_creme_treats": "奶油煮零食",
+  "block.fantasyfurniture.decorations.nordic.boiled_creme_treats.stacked": "堆积的奶油煮零食",
+  "block.fantasyfurniture.decorations.nordic.mead_bottles": "蜂蜜酒瓶",
+  "block.fantasyfurniture.decorations.nordic.mead_bottles.stacked": "堆积的蜂蜜酒瓶",
+  "block.fantasyfurniture.decorations.nordic.soul_gems_dark": "黑色灵魂宝石",
+  "block.fantasyfurniture.decorations.nordic.soul_gems_light": "灵魂宝石",
+  "block.fantasyfurniture.decorations.nordic.sweetrolls": "甜卷",
+  "block.fantasyfurniture.decorations.nordic.sweetrolls.stacked": "堆积的甜卷",
+  "block.fantasyfurniture.decorations.paper_stack": "纸叠",
+  "block.fantasyfurniture.decorations.tankards_empty": "酒杯",
+  "block.fantasyfurniture.decorations.tankards_empty.stacked": "堆积的大啤酒",
+  "block.fantasyfurniture.decorations.tankards_honeymead": "蜂蜜酒",
+  "block.fantasyfurniture.decorations.tankards_honeymead.stacked": "堆积的蜂蜜酒",
+  "block.fantasyfurniture.decorations.tankards_milk": "牛奶啤酒",
+  "block.fantasyfurniture.decorations.tankards_milk.stacked": "堆积的牛奶啤酒",
+  "block.fantasyfurniture.decorations.tankards_sweetberry": "甜莓酒",
+  "block.fantasyfurniture.decorations.tankards_sweetberry.stacked": "堆积的甜莓酒",
+  "block.fantasyfurniture.decorations.venthyr.chalices": "温西尔圣杯",
+  "block.fantasyfurniture.decorations.venthyr.chalices.stacked": "堆积的温西尔圣杯",
+  "block.fantasyfurniture.decorations.venthyr.food_0": "温西尔食品 1",
+  "block.fantasyfurniture.decorations.venthyr.food_1": "温西尔食品 2",
+  "block.fantasyfurniture.decorations.venthyr.platter": "温西尔拼盘",
+  "block.fantasyfurniture.decorations.venthyr.platter.stacked": "堆积的温西尔拼盘",
+  "block.fantasyfurniture.decorations.venthyr.tea_cups": "温西尔茶杯",
+  "block.fantasyfurniture.decorations.venthyr.tea_cups.stacked": "堆积的温西尔茶杯",
+  "block.fantasyfurniture.decorations.venthyr.tea_set": "温西尔茶具",
+  "block.fantasyfurniture.decorations.venthyr.tomes": "温西尔书籍",
+  "block.fantasyfurniture.decorations.venthyr.tomes.stacked": "堆积的温西尔书籍",
+  "block.fantasyfurniture.decorations.venthyr.widow_bloom": "寡妇花瓶",
+  "block.fantasyfurniture.dunmer.bed_double": "邓默双人床",
```

```suggestion
  "block.fantasyfurniture.dunmer.bed_double": "丹默双人床",
```
邓莫实应译为“丹莫“，以契合主流翻译，此问题请全文通查